src/Template/Events/submitted.ctp

```diff
@@ -84,6 +84,7 @@
                                 </tr>
                                 <tr>
                                     <td><strong>Details</strong></td>
+                                    <td>Submitted on: <?= $event->created ?></td>
```

Comment:
I think that adding a TD here will cause issues, as just that row will have a third column.

I believe we will need to add a TD to do each of the other rows, or make one of the TDs in the other rows have a colspan of 2